ibridges/data_operations.py

```diff
@@ -226,6 +226,8 @@ def _upload_collection(session: Session, local_path: Union[str, Path],
         Absolute irods destination path
     overwrite : bool
         If data already exists on iRODS, overwrite
+    ignore_err : bool
+        ignore failure on current item and continue to next one     
```

Comment:
I think we can give a bit more explanation here.
```suggestion
        If an error occurs during upload, and ignore_err is set to True, any errors encountered will be transformed into warnings and iBridges will continue to upload the remaining files. By default all errors will stop the process of uploading.
```